<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingForm</class>
 <widget class="QWidget" name="SettingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Python</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Executable:</string>
          </property>
          <property name="buddy">
           <cstring>textPython</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="textPython">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPython">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../visivo.qrc">
            <normaloff>:/icons/FILE_OPEN.bmp</normaloff>:/icons/FILE_OPEN.bmp</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBundlePython">
        <property name="text">
         <string>Use bundled environment</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tile</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Path</string>
          </property>
          <property name="buddy">
           <cstring>TileLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="TileLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="TilePushButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../visivo.qrc">
            <normaloff>:/icons/FILE_OPEN.bmp</normaloff>:/icons/FILE_OPEN.bmp</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Online</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="urlLineEdit">
          <property name="text">
           <string>http://visivo.oact.inaf.it/vialacteatiles/openlayers.html</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Glyph</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>max</string>
        </property>
        <property name="buddy">
         <cstring>glyphLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="glyphLineEdit">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>3D Visualization</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Max size (MB):</string>
          </property>
          <property name="buddy">
           <cstring>textDownscaleSize</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="textDownscaleSize">
          <property name="toolTip">
           <string>Set 0 to not downscale the cube</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="vlkbGroupBox">
     <property name="title">
      <string>VLKB</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>VLKB URL</string>
          </property>
          <property name="buddy">
           <cstring>vlkbUrl_lineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="vlkbUrl_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>TAP URL</string>
          </property>
          <property name="buddy">
           <cstring>tapUrl_lineEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="tapUrl_lineEdit">
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="statusLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="vlkbAuthLabel">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="vlkbAuthStatusLabel">
          <property name="text">
           <string>Not authenticated</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="vlkbLoginButton">
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vlkbLogoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>380</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkSearchOnImport">
        <property name="text">
         <string>Search when loading local data</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="caesarGroupBox">
     <property name="title">
      <string>CAESAR</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>API Endpoint</string>
          </property>
          <property name="buddy">
           <cstring>caesarEndpoint</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="caesarEndpoint">
          <property name="text">
           <string/>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="caesarAuthStatusLabel">
          <property name="text">
           <string>Not authenticated</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>10</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="caesarLoginButton">
          <property name="text">
           <string>Sign In</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="caesarLogoutButton">
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>380</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Working directory</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Path</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_2</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>~/VisIVODesktopTemp/tmp_download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="workdirButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../visivo.qrc">
          <normaloff>:/icons/FILE_OPEN.bmp</normaloff>:/icons/FILE_OPEN.bmp</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="OkPushButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>textPython</tabstop>
  <tabstop>btnPython</tabstop>
  <tabstop>checkBundlePython</tabstop>
  <tabstop>TileLineEdit</tabstop>
  <tabstop>TilePushButton</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>urlLineEdit</tabstop>
  <tabstop>glyphLineEdit</tabstop>
  <tabstop>textDownscaleSize</tabstop>
  <tabstop>vlkbUrl_lineEdit</tabstop>
  <tabstop>tapUrl_lineEdit</tabstop>
  <tabstop>vlkbLoginButton</tabstop>
  <tabstop>vlkbLogoutButton</tabstop>
  <tabstop>checkSearchOnImport</tabstop>
  <tabstop>caesarEndpoint</tabstop>
  <tabstop>caesarLoginButton</tabstop>
  <tabstop>caesarLogoutButton</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>workdirButton</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>OkPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../visivo.qrc"/>
 </resources>
 <connections/>
</ui>
